pax/simulation.py

```diff
@@ -465,33 +468,31 @@ def make_pax_event(self):
 
             # Did you want to superpose onto real noise samples?
             if self.config['real_noise_file']:
-                if noise_sample_mode != 'coherent':
+                #if noise_sample_mode != 'coherent':
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->